docs/input.md

```diff
@@ -110,15 +116,15 @@ There are a number of file types, which are loaded by default:
 | .dat  |  `numpy.array` | `vector` | `matrix`  | `number[][]` | 
 | .csv  |  `pandas.DataFrame` | `data.frame` |  `matrix` |  `number[][]` |
 
-#### asset
+#### `asset`
 
 The `type=asset` can be used to specify paths to files or entire folders that are copied unchanged to the `/in/` path of the tool container and thus made available to the tool for further processing. The parsing library never attempts to load and process these files, therefore the files are available in the tool exactly as they are.  
-Assets can be tool configurations, folders containing data, geometry files or all kinds of other files.
+Assets can be tool configurations, geometry files or all kinds of other files.
```

Comment:
"Assets are files that are available *as-is* in the container. Assets are neither Data nor parameters, but their dynamic nature might influence the tool execution. Hence, they are added as input to the tool."